tardis/spectrum/formal_integral_cuda.py

```diff
@@ -55,7 +55,7 @@ def cuda_formal_integral(
     z,
     shell_id,
 ):
-    """
+    r"""
```

Comment:
A docstring shouldn't need the raw string marker, should it?

Reply:
The docstring has this-
```
exp_tau : array(float64, 1d, C)
        $\exp{-tau}$ array to speed up computation
```

Perhaps because this is rendered on the docs, should I remove it? Ruff complains(
And I guess the tests will give the same warning as well.)-

```
tardis/spectrum/formal_integral_cuda.py:96:10: W605 [*] Invalid escape sequence: `\e`
Found 1 error.
```

Reply:
Fixed in [`cec4c0d` (#2745)](https://github.com/tardis-sn/tardis/pull/2745/commits/cec4c0d94190f5dafaa6c66ccb8a0c522da714a4)